### PR TITLE
Updated ont-tombo to version 1.4.

### PR DESCRIPTION
**Pull request by marcus1487**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/ont-tombo/meta.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ requirements:
   run:
     - python
     - setuptools >=18.0
```
Comment (bgruening) [inline recipes/ont-tombo/meta.yaml]: are you sure you need setuptools in run?

**Comment by marcus1487** [comment]
Probably don’t need setuptools. Is there a specifications page for extras/identifiers? I think I tried to add some a while back and couldn’t find them.

**Comment by marcus1487** [comment]
Found this [section](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#extra-section) but where do these extra bits end up?

**Comment by bgruening** [comment]
@marcus1487 for example here: https://github.com/bioconda/bioconda-recipes/blob/2c0ca47b12eba1fa06d0c3378670d3b50bfdfbf3/recipes/fastspar/meta.yaml#L53

**Pull request merged**